<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Tavtejs-MacBook-Air.local" failures="2" tests="21" name="calculatorTests.BasicOperations" time="66.458" errors="0" timestamp="2023-03-05T17:00:48 EST" skipped="11">
  <testcase classname="calculatorTests.BasicOperations" name="subtractionTest" time="6.632"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="negativeValueMulTest" time="6.218"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="multiplicationTest" time="5.872"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="infinityTest" time="6.300"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="decimalValueTest" time="6.092"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="additionTest" time="10.618"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="deleteTest" time="6.106">
    <failure message="Value mismatch test failed expected [92] but found [ERR]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Value mismatch test failed expected [92] but found [ERR]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at calculatorTests.BasicOperations.deleteTest(BasicOperations.java:169)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- deleteTest -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="bodmasTest" time="6.442">
    <failure message="Value mismatch test failed expected [15] but found [24]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Value mismatch test failed expected [15] but found [24]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at calculatorTests.BasicOperations.bodmasTest(BasicOperations.java:138)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- bodmasTest -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="errorAnalysisTest1" time="6.215"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="divisionTest" time="5.963"/>
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
  <testcase classname="calculatorTests.BasicOperations" name="readInputDisplay">
    <skipped/>
  </testcase> <!-- readInputDisplay -->
  <system-out/>
</testsuite> <!-- calculatorTests.BasicOperations -->
